### DISCUSSION TITLE
File Backup on Windows not working

### DISCUSSION THREAD

**Discussion by kayatechindia** [spatie/laravel-backup#1612] [Q&A]
I was backing up the public folder + the db.
DB get's zipped without a problem and also the file zipping doesn't throw an error.
But in the end the zipped archive only contains the db dump.

**Comment by erikn69** [comment]
It has no windows support

> This backup package requires PHP 8.0, with the [ZIP module](http://php.net/manual/en/book.zip.php) and Laravel 9.0 or higher. It's not compatible with Windows servers.

**Source:** https://spatie.be/docs/laravel-backup/v8/requirements
